src/components/InlineInput/InlineInput.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { InlineInput } from './InlineInput';
+
+export default { title: 'Components/InlineInput' };
+
+export const _InlineInput = {
+  render: () => (
+    <p>
+      <InlineInput value="1000000" />
+      <br />
+      <InlineInput value="0.00" disabled />
+      <br />
+      <InlineInput value="5.55" suffix=" USD" />
```

Comment:
if there is a risk that the developer does not add the space before the suffix, than it might me interesting to use a string litteral when you set the suffix value in the numeric format props : ``` suffix={` ${suffix}`}```

Reply:
yes but maybe the developer don't want the space in the suffix, I think it's better to let him the responsibility